### PR TITLE
difference also sorts columns

### PR DESCRIPTION
**Pull request by nonabelian**
The training set in the k fold CV has the original column order, but when picking out the test set using testing_columns, the columns get sorted by pd.index.difference. --> switched to drop.

### CODE REVIEW

**Comment by CamDavidsonPilon** [comment]
Wow great catch. I also discovered this in a different application, but totally ignored the implications here.

**Comment by CamDavidsonPilon** [comment]
Thanks @nonabelian!
